platform/server-config/src/main/java/org/motechproject/config/service/impl/ConfigurationServiceImpl.java

```diff
@@ -127,7 +125,6 @@ public void save(BootstrapConfig bootstrapConfig) {
     }
 
     @Override
-    @Caching(cacheable = {@Cacheable(value = SETTINGS_CACHE_NAME, key = "#root.methodName") })
     public MotechSettings getPlatformSettings() {
```

Comment:
please try to add transactional, because this method is using object from db. Probably we tried it and for some reason it didnt work out, however please just try